File: Admin_p/Admin_p.Web/Modules/electroDB/Payments/PaymentsColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace Admin_p.electroDB.Columns;

[ColumnsScript("electroDB.Payments")]
[BasedOnRow(typeof(PaymentsRow), CheckNames = true)]
public class PaymentsColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int PaymentId { get; set; }
    public string OrderStatus { get; set; }
    [EditLink]
    public string PaymentMethod { get; set; }
    public string PaymentStatus { get; set; }
    public DateTime TransactionDate { get; set; }
    public decimal Amount { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string TransactionId { get; set; }
}

File: Admin_p/Admin_p.Web/Modules/ServerTypes/electroDB/PaymentsColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { PaymentsRow } from "./PaymentsRow";

export interface PaymentsColumns {
    PaymentId: Column<PaymentsRow>;
    OrderStatus: Column<PaymentsRow>;
    PaymentMethod: Column<PaymentsRow>;
    PaymentStatus: Column<PaymentsRow>;
    TransactionDate: Column<PaymentsRow>;
    Amount: Column<PaymentsRow>;
    CreatedAt: Column<PaymentsRow>;
    UpdatedAt: Column<PaymentsRow>;
    TransactionId: Column<PaymentsRow>;
}

export class PaymentsColumns extends ColumnsBase<PaymentsRow> {
    static readonly columnsKey = 'electroDB.Payments';
    static readonly Fields = fieldsProxy<PaymentsColumns>();
}